### ISSUE TITLE
Does htmlText support multiple lines like text ?

### ISSUE THREAD

**Issue by zhouhao27** [mattdonnelly/MDHTMLLabel#23]
It's a great component. But I found the htmlText doesn't work the same way as the text does. It seems the multiple lines is supported but the auto shrink is not supported. Do I miss something? Thanks

**Comment by mattdonnelly** [comment]
It does support auto shrink but you need to specify the height of the label's frame (by setting it's rect or using Auto Layout) in order for it work

**Comment by zhouhao27** [comment]
If I need to set the frame manually, then it's not auto shrink. Am I right?

The AutoShrink for the **text** of label is to shrink based on the minimum scale factor or minimum font size. The height of the label won't be changed.

**Comment by mattdonnelly** [comment]
Yes, it doesn't affect the height of the label, but auto shrink won't work if the frame of label is flexible based on the content

**Comment by josluimg** [comment]
Yes. AutoShrink does not work for me, too. For example, putting html like <b>. Is a bug.
